src/App.js

```diff
@@ -1,16 +1,87 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList'
+import TodoForm from './components/TodoComponents/TodoForm'
+import './reset.css'
+import './index.css'
+//import './BobRoss.react'
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  // this is your world, you're the creator, find freedom on this canvas
+  constructor() {
+    super()
+    this.state = {
+      items: [],
```

Comment:
nice and clean state, digging it.. great to keep it simple and I would say I love the fact you're using comments but it seems to be bob ross quotes lol. Try to be more serious in your code as some day a potential employer will might randomly pick projects to look through.